app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/Cache.scala

```diff
@@ -9,17 +10,17 @@ import scalacache.CatsEffect.modes._
 
 object Cache extends LazyLogging {
 
-  val authorizationCacheFlags = Flags(Config.cache.authorizationCacheEnable,
```

Comment:
I think these are reversed - this should be setting `authenticationCacheFlags` not _authorization_

Reply:
Oh oops yes that's correct

Reply:
This is a very difficult file to get right